clang/test/CodeGenCXX/nocomdat-local-symbol.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clang_cc1 -emit-llvm -std=c++20 -triple x86_64-unknown-linux-gnu %s -o - | FileCheck %s
```

Comment:
Probably best to figure out where the primary test of the neighbor lines is and adds the extra test there.

Reply:
Done